src/definitions/elements/list.less

```diff
@@ -607,18 +594,18 @@ ul.ui.horizontal.bulleted.list li:first-child,
 .ui.horizontal.bulleted.list > .item:first-child {
   margin-left: 0;
 }
-ul.ui.horizontal.bulleted.list li::before,
-.ui.horizontal.bulleted.list > .item::before {
+ul.ui.horizontal.bulleted.list li:before,
```

Comment:
I am not complaining for a change, because the CSS2 versions of pseudo-elements will still work in every browser (who knows for how long), but instead of changing it here, we should discuss about a general ruleset.

I understand you have chosen this change/rule because you compared the whole codebase for less changes and i have to admit the CSS2/CSS3 :/:: variants are not consistent in the overall FUI codebase, but especially for this rule, my personal opinion is to use/keep/focce the CSS3 version using double colons instead, just because we do not support IE6/IE7/8 anymore which are the only browsers which don't understand the CSS3 format.

Reply:
Addressed in https://github.com/fomantic/Fomantic-UI/pull/580/commits/7f117500b0cc7fd5d0be768fe16b37bf8f097d4d
I agree to choose `::` over `:` since CSS2-only browsers are no longer supported.